lib/ecto/adapters/mysql.ex

```diff
@@ -115,6 +115,15 @@ defmodule Ecto.Adapters.MySQL do
     driver: :mariaex,
     migration_lock: "FOR UPDATE"
 
+  defmacro __before_compile__(env) do
+    message =
+      "#{inspect(__MODULE__)} is deprecated in favour of Ecto.Adapters.MyXQL " <>
+        "which uses the new MyXQL driver."
```

Comment:
I think we're lacking some actionable advice in this message - what should the user do to get rid of it?